Vagrantfile

```diff
@@ -119,11 +119,13 @@ fi
 
 echo "Creating bootstrap configuration"
 cat > /etc/metalk8s/bootstrap.yaml << EOF
-apiVersion: metalk8s.scality.com/v1alpha1
+apiVersion: metalk8s.scality.com/v1alpha2
```

Comment:
All those `s/v1alpha1/v1alpha2/` seems unrelated to the CA business, so they belong to a dedicated commit or PR.

Reply:
Huh? The CA is now in the configuration file, hence bumping the version.

Reply:
Ho, OK.

I thought it was related to the release of the alpha2.

Reply:
No no :) It's purely about the 'version' of this configuration file. In a sense, could've added the new field without bumping the version, we're nowhere near something 'frozen' anyway, but I think it made sense to set an example.

Either way, definitely not related to the MetalK8s version.